www/docs/customization/build.md

```diff
@@ -159,6 +159,17 @@ builds:
       - windows
 ```
 
+The binary name field supports [templating](/customization/templates/). The following build details are exposed:
+
+| Key     | Description                      |
+|---------|----------------------------------|
+| .Os     | `GOOS`                           |
+| .Arch   | `GOARCH`                         |
+| .Target | `GOARM`                          |
```

Comment:
this doesn't seem correct...

Reply:
that's true, I did some confusion here 🤦🏼 
fixing it now